client/go/internal/cli/cmd/test.go

```diff
@@ -198,6 +198,14 @@ func verify(step step, defaultCluster string, defaultParameters map[string]strin
 		method = "GET"
 	}
 
+	header := http.Header{}
+	for k, v := range step.Request.Headers {
+		header.Set(k, v)
+	}
+	if header.Get("Content-Type") == "" { // Set default if not specified by test
+		header.Set("Content-Type", "application/json") // TODO: Not guaranteed to be true ...
```

Comment:
```suggestion
		header.Set("Content-Type", "application/json")
```
Remove TODO that was really resolved now :)